### ISSUE TITLE
How to change the size of the photo crop area?

### ISSUE THREAD

**Issue by FillPower1** [ricardo-ch/react-easy-crop#108]
#### I'm sorry if I'm asking the wrong question
I want the cropping area to be smaller after uploading the photo. (50% less photo)
The documentation says to use "onMediaLoaded", but for some reason it does not work as it should, that is, the size of the cropping area changes only after uploading the photo again.
```
const [cropSize, setCropSize] = React.useState(null)
const onImageLoad = (img) => {
        const circleWidth = 50
        const circleSize = img.width < img.height
            ? (img.width * circleWidth) / 100
            : (img.height * circleWidth) / 100
        setCropSize({ width: circleSize, height: circleSize })
    }

<Cropper
   classes={{ containerClassName: styles.container }}
    image={`http://localhost:3000/${src}`}
     crop={crop}
     zoom={zoom}
     aspect={1}
     cropShape="round"
     showGrid={false}
     onCropChange={onCropChange}
     onCropComplete={onCropComplete}
     onZoomChange={setZoom}
     cropSize={cropSize}
     onMediaLoaded={onImageLoad}
/>                                
````

https://codesandbox.io/s/react-easy-crop-custom-image-demo-8gpbz

If you upload the photo again, you will see how the cropping area changes

**Comment by ValentinH** [comment]
Hi,

Indeed, `cropSize` is not watched for change. You would need to add it in the `componentDidUpdate()` method here: https://github.com/ricardo-ch/react-easy-crop/blob/master/src/index.tsx#L118

You can submit a PR if this solves your issue :)

**Comment by FillPower1** [comment]
Understood, I will understand and send PR)

**Comment by ValentinH** [comment]
Released under 2.1.1 🎉
